raml/api-writer.raml

```diff
@@ -0,0 +1,76 @@
+#%RAML 0.8
+title: Write Only Runbook API Service
+version: v1
+mediaType: application/json
+baseUri: /api/{version}
+schemas:
+  - 2xx: !include schemas/2xx.json
+  - 4xx: !include schemas/4xx.json
+
+/region:
+  /{region}:
+    uriParameters:
+      region:
+        description: Region name which is region identifier
+        type: string
+
+    /runbooks:
+      post:
+        description: Create a new runbook
+        body:
+          application/json:
+            formParameters: !include forms/runbook.yaml
+        responses:
+          201:
+            body:
+              schema: 2xx
+          400:
+            body:
+              schema: 4xx
+
+      /{runbook_id}:
+        uriParameters:
+          runbook_id:
+            description: Id of a particular runbook
+            type: string
+        put:
```

Comment:
> Normally, all PUT requests are made for the same URL
well, no. This url allows altering a specific runbook, so you already know the resource id.
as per RFC 
> The PUT method requests that the state of the target resource be
>    created or replaced with the state defined by the representation
>    enclosed in the request message payload 
so a PUT request to .../runbooks should replace **the entire** collection with another collection. 
https://en.wikipedia.org/wiki/Representational_state_transfer#Relationship_between_URL_and_HTTP_methods this article gives a nice overview on how restfull api's usually correspond to http methods
or this article http://restful-api-design.readthedocs.io/en/latest/methods.html#standard-methods
Usually PUT is used on resource-scope, not collection